android/clientlib-rxjava/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
     compileSdk 32
 
     defaultConfig {
-        minSdk 23
+        minSdk 24
```

Comment:
What necessitated this? It's not the end of the world, but it would be convenient to keep all the minSdk versions of MWA in lockstep.

Reply:
Here is my thought on this matter. We are so close to API 24, I think we should just make the jump. That's still Android 7.1 which is very, very old. Saga is 6 OS versions ahead of that. If the community _needs_ lower API support, they can fork and make it work for themselves. Thoughts?